src/ex_cmds2.c

```diff
@@ -198,6 +205,7 @@ dialog_changed(
 			|| (cmdmod.cmod_flags & CMOD_BROWSE)
 #endif
 			)
+		    && !bt_dontwrite(buf2)
```

Comment:
This fixes an issue when you do `:confirm qall` with a live terminal, and it complains about not having autocmds associated with it. It still works but this makes sure we don't hit that code path